packages/runtime/src/templates/server.ts

```diff
@@ -23,6 +23,10 @@ const getNetlifyNextServer = (NextServer: NextServerType) => {
     private netlifyConfig: NetlifyConfig
     private netlifyPrerenderManifest: PrerenderManifest
 
+    public getAppRouterReactVersion(): string {
```

Comment:
Added this public method so we can get the React version to set `process.env.__NEXT_PRIVATE_PREBUNDLED_REACT` if the request fails with `[ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './ server.edge' is not defined`